dev-vcs/qgit/qgit-2.9.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A Qt-based git GUI viewer"
+DESCRIPTION="With qgit you will be able to browse revisions history,\
+view patch content and changed files, graphically following different \
+development branches."
+HOMEPAGE="https://github.com/tibirna/qgit"
+COPYRIGHT="2011-2019 Cristian Tibirna"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/tibirna/qgit/archive/qgit-$portVersion.tar.gz"
+CHECKSUM_SHA256="69788efda7d434d1bc094bb414bd92c269dc7894326320634500b05d63c008e8"
+SOURCE_DIR="qgit-qgit-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qgit$secondaryArchSuffix = $portVersion
+	cmd:qgit$secondaryArchSuffix
+	app:QGit
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:git
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	qthaikuplugins$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:qmake$secondaryArchSuffix >= 5
+	"
+
+defineDebugInfoPackage qgit$secondaryArchSuffix \
+	$appsDir/QGit
+
+BUILD()
+{
+	qmake qgit.pro
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p {$appsDir,$binDir}
+	cp bin/qgit $appsDir
+	mv $appsDir/qgit $appsDir/QGit
+	ln -s $appsDir/QGit $binDir/qgit
```

Comment:
Any specific reason you want this in $binDir?

Reply:
Well, this app is a GUI app but can also work in the terminal as a command-line app, so I put it in `$binDir` where all the command-line apps belong.

Reply:
You can use QGit from Terminal if it's installed in $appsDir :)

Reply:
Just checked, no options when launching from Terminal, so no use of it linking to $binDir

Reply:
This is from QGit's [official page](https://github.com/tibirna/qgit)
> 
> Run qgit from a git working directory, command line arguments are filtered by git log. Some examples:
> qgit --no-merges
> qgit v2.6.18.. include/scsi drivers/scsi
> qgit --since="2 weeks ago" -- kernel/
> qgit -r --name-status release..test
> If qgit is launched without arguments or if you change archive with open menu, a dialog for range select is shown. You can select top and bottom rev tags from the list or paste a specific revision. Values are passed to git log to narrow data loading to chosen revisions.
> As a special command-line argument, --view-file foo or --view-file=foo is not passed to git log, but opens the file foo in the QGit file viewer, showing the history and the annotated (blame) view for the file.

Reply:
I think it is some command line options? If it is not, then I will remove the links and also `cmd:qgit` from the PROVIDES section.

Reply:
I think the options (haven't checked those) can be used with QGit the same as with qgit, need to check

Reply:
Even when linking it to $prefix/bin it launches the GUI here:
![QGit](https://user-images.githubusercontent.com/16057090/72819769-200ccb80-3c6e-11ea-9e67-eb07a588bf48.png)

Reply:
I see it. So there's no point in including cmd:qgit.